loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowParametersDeserializer.java

```diff
@@ -84,4 +94,25 @@ public LoadFlowParameters deserialize(JsonParser parser, DeserializationContext
         return parameters;
     }
 
+    private static void assertLessThanOrEqualToReferenceVersion(String tag, String version, String referenceVersion) {
+        if (version == null) {
+            throw new PowsyblException("LoadflowParameters. Unexpected null pointer for version");
+        } else if (version.compareTo(referenceVersion) > 0) {
+            String exception = String.format(
+                "LoadflowParameters. Tag: %s is not only valid for LoadflowParameters version %s. LoadFlowParameters version should be <= %s %n",
+                tag, version, referenceVersion);
+            throw new PowsyblException(exception);
+        }
+    }
+
+    private static void assertGreaterThanReferenceVersion(String tag, String version, String referenceVersion) {
+        if (version == null) {
+            throw new PowsyblException("LoadflowParameters. Unexpected null pointer for version");
+        } else if (version.compareTo(referenceVersion) <= 0) {
+            String exception = String.format(
+                "LoadflowParameters. Tag: %s is not only valid for LoadflowParameters version %s. LoadFlowParameters version should be > %s %n",
+                tag, version, referenceVersion);
+            throw new PowsyblException(exception);
+        }
+    }
```

Comment:
Rather than putting these methods here, put them in `JsonUtil`, they could be used in other versioned JSON serializers/deserializers.

Reply:
Done